R/check-cran.sh

```diff
@@ -36,11 +36,27 @@ if [ ! -z "$R_HOME" ]
 fi
 echo "USING R_HOME = $R_HOME"
 
-# Build the latest docs
-$FWDIR/create-docs.sh
+# Build the latest docs, but not vignettes, which is built with the package next
+NO_VIGNETTES=1 $FWDIR/create-docs.sh
 
-# Build a zip file containing the source package
-"$R_SCRIPT_PATH/"R CMD build $FWDIR/pkg
+# Build source package with vignettes
+SPARK_HOME="$(cd "`dirname "$0"`"/..; pwd)"
```

Comment:
what that command is doing is
1. get the dirname of the currently running file, check-cran.sh (`dirname "$0"`)
2. cd to one level up (`cd x/..`)
3. print current directory `pwd`
4. set that to SPARK_HOME

so it shouldn't matter what the current directory is but only with where the check-cran.sh is found?

I'm running this on Ubuntu, it's possible if you are running on Mac the behavior is different somehow?